Dockerfile

```diff
@@ -157,6 +157,7 @@ RUN set -ex \
     && apk add --virtual .drupal-phpexts-rundeps $runDeps \
     && git clone -b NON_BLOCKING_IO_php7 https://github.com/websupport-sk/pecl-memcache.git \
     && cd pecl-memcache \
+    && git checkout 4a9e4ab0d12150805feca3012854de9fd4e5a721 \
```

Comment:
ugh

Reply:
Right?   I hate it more because that memcache library is barely supported anymore and thats the only place I can get one that works with D8.